### ISSUE TITLE
Click on room on indoor map opens room detail page

### ISSUE THREAD

**Issue by anonym-HPI** [hpi-swt2/compass-portal-blue#216]
As a user I want to see the detail page of a room when clicking on it on the map, because I don't want to have to search for a room when I know which room I want to see the detail page of.

**Comment by oleschl** [comment]
It already is kinda working. Two issues:

- how should we map between the leaflet polygon and the database entry of a room (thats why i am only searching and not showing the detail page)
- multiple click events eg. place marker vs click on room

https://user-images.githubusercontent.com/49571011/152567751-2dc0dbdc-897e-4403-9fb1-81981b85289f.mp4

**Comment by oleschl** [comment]
@anonym-HPI could you also have a look at the issue here? Right now the first click on the map places a marker and the second places another one and you can start a route. However, the user probably only wanted to see the detail page and get a route. You can see the issue in the video above: the first click displays the search result but second opens the route window instead of the detail page.

**Comment by anonym-HPI** [comment]
@oleschl correct, it was intended to see the detail page of the room. I am thinking how this interfears with the routing, as when you want to navigate to a room it should probably also be clickable. 
Can you remove the first pin with clicking again on it? So that only the second is marked and shown in the left bottom corner? (like in the beginning of the clip).

**Comment by anonym-HPI** [comment]
It will exist (in development) a script which will fill polygons into the database, as a site note.

It would probably be good to have in every Geojson a room id (unique over the complete project, could be just the id of the room in the database), that links to the room in the database and could then be used to open it.

What do you think @oleschl ?
